ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -0,0 +1,326 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            "common/python"))
+    import parsing_lib
+
+_AGE = "FaixaEtaria"
+_GENDER = "Sexo"
+_MUNICIPALITY = "Municipio"
+_DATE_CONFIRMED = "DataNotificacao"
+_DATE_DEATH = "DataObito"
+_CLASSIFICATION = "Classificacao"
+_OUTCOME = "Evolucao"
+_LUNG = "ComorbidadePulmao"
+_KIDNEY = "ComorbidadeRenal"
+_DIABETES = "ComorbidadeDiabetes"
+_OBESITY = "ComorbidadeObesidade"
+_CARDIOVASCULAR = "ComorbidadeCardio"
+_SMOKER = "ComorbidadeTabagismo"
+_PCR = "DataColeta_RT_PCR"
+_SEROLOGICAL_METHOD1 = "DataColetaTesteRapido"
+_SEROLOGICAL_METHOD2 = "DataColetaSorologia"
+_SEROLOGICAL_METHOD3 = "DataColetaSorologiaIGG"
+_TEST_CLASS = "CriterioConfirmacao"
+_ETHNICITY = "RacaCor"
+_NEIGHBOURHOOD = "Bairro"
+_FEVER = "Febre"
+_DIFFICULTY_BREATHING = "DificuldadeRespiratoria"
+_COUGH = "Tosse"
+_SORE_THROAT = "DorGarganta"
+_DIARRHOEA = "Diarreia"
+_HEADACHE = "Cefaleia"
+_CORYZA = "Coriza"
+_HOSPITALIZED = "FicouInternado"
+_INTERNAL_TRAVEL = "ViagemBrasil"
+_INTERNATIONAL_TRAVEL = "ViagemInternacional"
+_HEALTHCARE_PROFFESIONAL = "ProfissionalSaude"
+_HOMELESS = "MoradorDeRua"
+_DISABILITY = "PossuiDeficiencia"
+_SCHOOLING = "Escolaridade"
+
+
+_COMORBIDITIES_MAP = {
+    "ComorbidadePulmao": "lung disease",
+    "ComorbidadeRenal": "kidney disease",
+    "ComorbidadeDiabetes": "diabetes mellitus",
+    "ComorbidadeObesidade": "obesity",
+    "ComorbidadeCardio": "cardiovascular system disease"
+}
+
+
+_EDUCATION_MAP = {
+    "1ª a 4ª série incompleta do EF (antigo primário ou 1º grau)": "First four years of elementary school not completed",
+    "Ensino fundamental completo (antigo ginásio ou 1º grau) ": "Elementary school completed",
+    "Ensino médio completo (antigo colegial ou 2º grau ) ": "High school completed",
+    "5ª à 8ª série incompleta do EF (antigo ginásio ou 1º grau)": "Years 5 to 8 of elementary school not completed",
+    "Ensino médio incompleto (antigo colegial ou 2º grau )": "High school not completed",
+    "Educação superior incompleta ": "Higher education not completed",
+    "Educação superior completa": "Higher education completed",
+    "Analfabeto": "Analphabet",
+    "4ª série completa do EF (antigo primário ou 1º grau)": "Fourth year of elementary school completed"
+}
+
+
+_NONE_TYPES = set(["Não", "Ignorado", "-", "Não Informado", "Não se aplica", None])
```

Comment:
I quite like this solution. Very readable lower down. Let's keep our eyes out for ramifications around including None in that list - if it shows up in an unexpected place, the behavior can be pretty weird.